src/components/AppNavigation/Settings/SettingsImportSection.vue

```diff
@@ -171,7 +171,7 @@ export default {
 				// https://developer.mozilla.org/en-US/docs/Web/API/File/type
 				// "Uncommon" file-extensions will result in an empty type
 				// and apparently Microsoft considers calendar files to be "uncommon"
-				if (type === '') {
+				if (type === '' || type.startsWith('application/x-extension-')) {
```

Comment:
Another way would just be to check if the type isn't already one of `application/calendar+xml`, `application/calendar+json`, `text/csv` or `text/calendar`, since this is not reliable.

Reply:
Done